Sources/SnapshotTesting/Snapshotting/NSAttributedString.swift

```diff
@@ -0,0 +1,37 @@
+#if canImport(Cocoa)
+import Cocoa
+
+extension Snapshotting where Value == NSAttributedString, Format == NSImage {
+  public static func image(precision: Float = 1, maxWidth: CGFloat) -> Snapshotting {
+    return Snapshotting<NSView, NSImage>.image(precision: precision).pullback { attributedString in
+      let label = NSTextField()
+      label.attributedStringValue = attributedString
+      label.backgroundColor = .white
+      label.isBezeled = false
+      label.isEditable = false
+      label.preferredMaxLayoutWidth = maxWidth
+      label.frame.size = label.fittingSize
+      return label
+    }
+  }
+}
+#elseif canImport(UIKit)
+import UIKit
+
+extension Snapshotting where Value == NSAttributedString, Format == UIImage {
+  public static func image(precision: Float = 1, maxWidth: CGFloat) -> Snapshotting {
```

Comment:
Is `maxWidth` right?

Reply:
I think just `width` would be fine since the image probably will be that exact width.